desktop/api/src/main/java/org/datacleaner/util/WidgetUtils.java

```diff
@@ -75,34 +75,24 @@
 
     private static final Logger logger = LoggerFactory.getLogger(WidgetUtils.class);
 
-    public static final float FONT_SIZE_SMALL = 11f;
+    private static final WidgetScreenResolutionAdjuster adjuster = WidgetScreenResolutionAdjuster.get();
+
+    private static final int PIX_1 = adjuster.adjust(1);
+    private static final int PIX_2 = adjuster.adjust(2);
+    private static final int PIX_3 = adjuster.adjust(3);
+    private static final int PIX_4 = adjuster.adjust(4);
+    private static final int PIX_6 = adjuster.adjust(6);
+    private static final int PIX_9 = adjuster.adjust(9);
+    private static final int PIX_10 = adjuster.adjust(10);
+    private static final int PIX_17 = adjuster.adjust(17);
```

Comment:
This whole block is pretty nasty (and quite inconsistently used)

If you really want the indirection, how about a call instead?

```java
int pix(int in) {
    return adjuster.adjust(in);
}

public static final Object BORDER_MENU_ITEM = new EmptyBorder(pix(2), pix(2), pix(2), pix(2));
```

The JIT compiler should optimize frequent calls pretty well.

Reply:
I almost feel like adding that to the adjuster then, as a static method that one can do a static import of.

Reply:
Oh well, the name `pix` is a bit too internal for that. So I did what you suggested :-)